### PR TITLE
fix: Change Password Button not working in any language other than English language mode

### LINKED ISSUE #1756: Change Password Button not working in any language other than English language mode
**Summary:**

Change Password Button not working in any language other than English language mode

**Steps to reproduce:**

Watch the video

**Expected behaviour:**

Change Password Button should work in every language

**Observed behaviour:**

Change Password Button should does not work in any language other than English

**Device and Android version:**

Any

**Screenshots:**

https://user-images.githubusercontent.com/54978105/109628179-73876380-7b68-11eb-8966-0b8334acb797.mp4

### PR DESCRIPTION
**Pull request by kingbass01**
Fixes #1756 

Please Add Screenshots If there are any UI changes.

https://user-images.githubusercontent.com/85404940/225634332-1abefb1e-f29e-4dbf-b8aa-2d348426ecf6.mp4

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by kingbass01** [comment]
Hey @jawidMuhammadi , can you please review this?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -153,6 +153,7 @@
     <string name="no">No</string>
     <string name="found">found</string>x
 
+    <string name="password_key">Password</string>
```
Comment (jawidMuhammadi) [inline app/src/main/res/values/strings.xml]: As this string will act as a key then it should not be translated to other languages, so please add` translatable="false"`

**Review by jawidMuhammadi — CHANGES_REQUESTED** [review]

**Comment by jawidMuhammadi** [comment]
The title of the PR should start as `fix: change Password Button ...`

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (kingbass01) [reply app/src/main/res/values/strings.xml]: Sorry for not paying attention to that, changes done.

**Review by jawidMuhammadi — APPROVED** [review]
Looks good to me, good job! @kingbass01

**Comment by kingbass01** [comment]
Hey @jawidMuhammadi, can you merge this PR?

**Pull request merged**